src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -0,0 +1,196 @@
+---
+title: "Hands-on with Portals: seamless navigations on the Web"
+author: uskay
+hero: hero.png
+alt: A logo of Portals
+description: |
+  An article on explaining how to use the new HTML element: Portals
+tags:
+  - portals
+  - UX
+---
+
+We’ve long stressed the importance of web performance. While it is tempting to focus on individual pages, it is also critical to maintain a good user experience as your users navigate across your pages. A new upcoming web platform API called [Portals](https://github.com/WICG/portals) can help you with that.
+
+![A gif of a portals demo](hero.gif)
+
+# What Portals enables
+Single Page Applications with an [App Shell model](https://developers.google.com/web/fundamentals/architecture/app-shell) are one way of achieving seamless navigations. Having a static shell on the page, like the header, footer and side nav, and dynamically changing the content with a transition provides continuity to the user experience in ways that a classic navigation experience in Multiple Page Applications can’t. In particular, the navigation experience on Multiple Page Applications is plagued by frequent flashes of blank or unfinished paints. While SPA result in a better user experience, there are many reasons why you might not want to choose SPA as your frontend architecture. Maybe you don’t have the necessary expertise in your team, or you might just want to simplify the front-end and strategically select the more scalable and productivity friendly MPA architecture. The Web is open to any types of websites and each one of them should have a good way to satisfy the users with liquid smooth experience.
+
+Portals is a new HTML element that can help you keep your front-end simple while allowing seamless navigations with custom transitions. The best part is that Portals is quite easy to work with. Think of it as an iframe in that it allows for embedding, but unlike an iframe it also comes with a feature to directly navigate into its content. Seeing is believing: please first check out what we showcased in Chrome Dev Summit last year.
```

Comment:
I think we are using the word "Portals" as a proper noun. e.g. 
> Portals is a very early proposal
https://github.com/WICG/portals/blob/master/README.md

Reply:
I'm wondering if the usage in your GitHub example is because it's referring to the portals proposal (singular), and not portals themselves. As someone unfamiliar with the background and reading the article for the first time it looked like a grammatical error.

If it's referring to a new HTML element it feels a bit odd to refer to it as  a proper noun. Like saying "Buttons is" or "Iframes is". In the youtube video one of the speakers says, "So, what are portals? Portals are like iframes."

Reply:
Makes sense. I'll fix it.

Reply:
I changed this as well but now I'm thinking I'm wrong

> Whereas the improved experience with Portals **were** pre-rendering the next chapter

In this case, the subject would be experience and I should have kept it to **was**?